packages/apps/human-app/server/src/integrations/external-api/external-api.mapper.ts

```diff
@@ -0,0 +1,65 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { Injectable } from '@nestjs/common';
+import {
+  CamelCaseNamingConvention,
+  createMap,
+  forMember,
+  mapFrom,
+  Mapper,
+  namingConventions,
+  SnakeCaseNamingConvention,
+} from '@automapper/core';
+import {
+  JobAssignmentData,
+  JobAssignmentParams,
+  JobsFetchParams,
+  JobsFetchParamsData,
+} from '../../modules/job-assignment/interfaces/job-assignment.interface';
+import {
+  JobsDiscoveryParams,
+  JobsDiscoveryParamsData,
+} from '../../modules/jobs-discovery/interfaces/jobs-discovery.interface';
+
+@Injectable()
+export class ExternalApiProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper: Mapper) => {
+      createMap(
+        mapper,
+        JobAssignmentParams,
+        JobAssignmentData,
+        namingConventions({
+          source: new CamelCaseNamingConvention(),
+          destination: new SnakeCaseNamingConvention(),
+        }),
+      );
+      createMap(
+        mapper,
+        JobsFetchParams,
+        JobsFetchParamsData,
+        namingConventions({
+          source: new CamelCaseNamingConvention(),
+          destination: new SnakeCaseNamingConvention(),
+        }),
+      );
+      createMap(
+        mapper,
+        JobsDiscoveryParams,
+        JobsDiscoveryParamsData,
+        // Automapper has problem with mapping arrays, thus explicit conversion
```

Comment:
// FIXME: `https://github.com/nartc/mapper/issues/583`

Reply:
this is not this case